app/models/user.rb

```diff
@@ -14,4 +14,6 @@ class User < ApplicationRecord
   has_many :comments
 
   has_many :user_positions
+
+  mount_uploader :remote_photo_url, PhotoUploader
```

Comment:
it should be changed to :photo instead of :remote_photo_url

Reply:
user story made for this refactoring